src/dataframe/dataframe.jl

```diff
@@ -689,7 +663,7 @@ function deleterows!(df::DataFrame, ind::AbstractVector{Int})
     idf = 1
     iind = 1
     ikeep = 1
-    keep = Vector{Int}(n - length(ind2))
+    keep = Array{Int}(n-length(ind2))
```

Comment:
?

Reply:
This was `Array` before the fork, and has been changed to `Vector` in DataFrames since then. I've changed the merge commit to use `Vector` since that's slightly better.